README.md

```diff
@@ -17,6 +17,9 @@ Note: to get this repository, you will need to download Lean's mathematical libr
 ### Get the Repository using VSCode
 
 * Open Visual Studio Code.
+
+* If no file is open, create a new empty text file (`File ... > New Text File`).
```

Comment:
Is this step necessary?

Reply:
For me it was, I had just closed a folder in VS Code before starting this walk-through, and there was no ∀ symbol. It only became visible when I first did CTRL+N.